tests/framework/db/CommandTest.php

```diff
@@ -1538,13 +1538,13 @@ public function testBindValuesSupportsEnums()
 		    $db = $this->getConnection();
 		    $command = $db->createCommand();
 
-		    $command->setSql('SELECT :p1')->bindValues([':p1' => enums\Status::ACTIVE]);
+		    $command->setSql('SELECT :p1')->bindValues([':p1' => enums\Status::Active]);
 		    $this->assertSame('ACTIVE', $command->params[':p1']);
```

Comment:
The expected value should be renamed also.

Reply:
@briedis can you prepare a PR to address this please. @Izumi-kun thanks for spotting and pointing this out

Reply:
Sorry about that. Here's the PR: https://github.com/yiisoft/yii2/pull/20296

Reply:
No need to be sorry. It is OK.
Thanks for PR